app/shop.py

```diff
@@ -0,0 +1,76 @@
+from math import dist
+
+
+class Shop:
+    def __init__(self,
+                 data: dict,
+                 customer: dict,
+                 fuel_price: int | float) -> None:
+        self.data = data
+        self.customer = customer
+        self.fuel = fuel_price
+
+    @staticmethod
+    def cost_of_trip(distance: int | float,
+                     fuel_cons: int | float,
+                     fuel_cost: int | float) -> int | float:
+        return (fuel_cons * (distance * 0.01)) * fuel_cost
+
+    @staticmethod
+    def check_money(customer_cart: dict,
+                    shop_price: dict) -> int | float:
+        _sum = 0
+        for element in customer_cart:
+            _sum += customer_cart[element] * shop_price[element]
+        return _sum
+
+    def iter_shops(self) -> tuple:
+        totals_of_shop = []
+        best_cost = []
```

Comment:
```suggestion
        minimal_price, best_shop = None, None
```
Use variables instead of list